sandbox.go

```diff
@@ -0,0 +1,488 @@
+package libnetwork
+
+import (
+	"container/heap"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/libnetwork/sandbox"
+	"github.com/docker/libnetwork/types"
+)
+
+// Sandbox provides the control over the network container entity. It is a one to one mapping with the container.
+type Sandbox interface {
+	// Info returns the info regarding this container.
+	Info() SandboxInfo
+	// Delete destroys this container after detaching it from all connected endpoints.
+	Delete() error
+}
+
+// SandboxInfo provides an interface to retrieve the info about the sandbox.
+type SandboxInfo interface {
```

Comment:
Actually Endpoint has EndpointInfo as sub interface. I followed the same here. Maybe ID() and Key() could be part of Sandbox but more detailed info like labels and stats I feel they belong to SandboxInfo.